app/metrics.yaml

```diff
@@ -441,6 +441,52 @@ context_menu:
       - fenix-core@mozilla.com
     expires: "2020-10-01"
 
+login_dialog:
+  displayed:
+    type: event
+    description: |
+      The login dialog prompt was displayed
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/9730
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/13050
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-10-01"
```

Comment:
Nit: We shouldn't batch all the telemetry reviews to be the same date, default should be 6mo from now, so that puts us around feb/march.